gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/store/GemFireContainer.java

```diff
@@ -3350,7 +3350,7 @@ public final void insertMultipleRows(final ArrayList<Object> rows,
       observer.insertMultipleRowsBeingInvoked(numRows);
       observer.invokeCacheCloseAtMultipleInsert();
     }
-    if (GemFireXDUtils.TraceConglomUpdate | GemFireXDUtils.TraceQuery) {
+    if (GemFireXDUtils.TraceConglomUpdate | GemFireXDUtils.TraceQuery && false) {
```

Comment:
Revert. Can take out the TraceQuery part if you think its generating too many logs with TraceQuery enabled.